standard_reports/item-usage/latest/convert_data_js/src/test/output.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Also moved this one, and fully reflect output data, plus checking sort with item names (also note capitilisation made to test, since 'I' < 'i'